src/checkbox-group/index.tsx

```diff
@@ -18,6 +18,8 @@ export interface CheckboxGroupProperties {
 	initialValue?: string[];
 	/** A controlled value for the checkbox group */
 	value?: string[];
+	/** Disabled all inputs within this group */
+	disabled?: boolean;
```

Comment:
Do you think we should pass the disabled value down to the custom renderer as well so that the user can decide to do something with that or nah?

Reply:
Yes! Good catch.